### PR TITLE
xg boost updated 

### PR DESCRIPTION
**Pull request by dikshajoshi18**
I have used a class-based format.

### CODE REVIEW

File: XGBoost/XGBoost (1).py [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# XGBoost 
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
```
Comment (ashwani-rathee) [inline XGBoost/XGBoost (1).py:5]: you don't need to import anything other than numpy,pandas for this class type implementation..

File: XGBoost/XGBoost (1).py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+import pandas as pd
+import matplotlib.pyplot as plt
+import IPython
+from IPython import get_ipython
+
+
+# Consider the following data.
+class XGB:
+    def __init__(self):
+        age = [9, 23, 24, 27, 45, 56, 62, 71, 76, 82, 84, 95, 79, 29, 34, 35, 49]
```
Comment (ashwani-rathee) [inline XGBoost/XGBoost (1).py:13]: You have used fixed data,please remove it and make it work for a general case

File: XGBoost/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+Here is the Mathematical representation of a boosting classifier<br>
+                                               ![formula](https://user-images.githubusercontent.com/58946705/111885101-a7c5a580-89eb-11eb-8dee-fd71bc899a2d.jpg)
+
+Here, **T** is the no. of classifiers, **ft** is a weak learner and **x** is the data input. It returns the class
+
+ It can be depicted by the below diagram:
+ ![flowchart](https://user-images.githubusercontent.com/58946705/111885187-2a4e6500-89ec-11eb-919d-a9f260f2487c.jpg)
+
+
+### XGBoost
```
Comment (ashwani-rathee) [inline XGBoost/README.md:18]: This needs to have algorithm,advantages,disadvantages and references

**Review by ashwani-rathee — CHANGES_REQUESTED** [review]

File: XGBoost/XGBoost (1).py (hunk 1 above)
Comment (dikshajoshi18) [reply XGBoost/XGBoost (1).py:5]: done

File: XGBoost/XGBoost (1).py (hunk 2 above)
Comment (dikshajoshi18) [reply XGBoost/XGBoost (1).py:13]: done

**Review by dikshajoshi18 — COMMENTED** [review]
@ashwani-rathee  I have made all the required changes. Kindly review them

File: XGBoost/README.md (hunk 3 above)
Comment (dikshajoshi18) [reply XGBoost/README.md]: done

File: XGBoost/XGBoost (1).py [hunk 4]
```diff
@@ -0,0 +1,90 @@
+# XGBoost
+
+import numpy as np
+import pandas as pd
+
+# Self defined Class for XGBoost
+class XGB:
+    def __init__(self, X, Y):
+        self.df = pd.DataFrame(columns=['age', 'fees'])
```
Comment (geekquad) [inline XGBoost/XGBoost (1).py:9]: You have used parameters for fixed data, please remove it and make it work for a general case.

**Comment by yukti845** [comment]
@dikshajoshi18  Please ass the proposed changes and resolve the conflicts.

**Comment by geekquad** [comment]
Closing this issue.

**Pull request closed without merging**